fsharp-backend/tests/FuzzTests/OCamlInterop.FuzzTests.fs

```diff
@@ -60,27 +60,28 @@ let isInteroperable
       false
   with
   | e ->
-    print $"Cause exception while fuzzing {e}"
+    print $"Exception while fuzzing {e}"
     reraise ()
 
 type Generator =
+  static member SafeString() : Arbitrary<string> = Arb.fromGen (G.string ())
+
   static member Expr() =
     Arb.Default.Derive()
     |> Arb.filter (function
-      // characters are not yet supported in OCaml
+      // characters are not supported in OCaml
+      // CLEANUP can be removed once OCaml gone
```

Comment:
maybe this is redundant; the `FuzzTests` project probably needs a full once-over once OCaml is "gone"